modules/cli/cmds/apply_test.go

```diff
@@ -0,0 +1,97 @@
+package cmds
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/rancher/fleet/modules/cli/apply"
+	"github.com/rancher/fleet/pkg/bundlereader"
+	"sigs.k8s.io/yaml"
```

Comment:
:thinking: Do we have rules/standards in terms of ordering imports?
eg.
```
<standard lib imports>
<blank line>
<third-party imports>
<blank line>
<project imports>
```

```suggestion
	"errors"
	"testing"

	"github.com/google/go-cmp/cmp"
	"sigs.k8s.io/yaml"
	
	"github.com/rancher/fleet/modules/cli/apply"
	"github.com/rancher/fleet/pkg/bundlereader"
```